content/docs/fields/number.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Number Field
+next: /docs/fields/textarea
+consumes:
+  - file: /packages/tinacms/src/plugins/fields/NumberFieldPlugin.tsx
+    details: Shows text field interface and config options
+  - file: /packages/@tinacms/fields/src/NumberField.ts
+    details: Shows text field interface and config options
+---
+
+The `number` field represents a number input.
+
+<!-- TODO: Provide image of number field -->
+
+## Definition
+
+Below is an example of how a `number` field could be defined in a Gatsby remark form. [Read more on passing in form field options](/docs/gatsby/markdown#customizing-remark-forms).
+
+```javascript
+const BlogPostForm = {
+  fields: [
+    {
+      name: 'rawFrontmatter.weight',
+      component: 'number',
+      label: 'Weight',
+      description: 'Enter a weight for post sorting',
+    },
+    // ...
+  ],
+}
+```
+
+## Options
+
+- `name`: The path to some value in the data being edited.
+- `component`: The name of the React component that should be used to edit this field. Available field component types are [defined here](/docs/concepts/fields#field-types)
+- `label`: A human readable label for the field. This label displays in the sidebar and is optional. If no label is provided, the sidebar will default to the name.
+- `description`: An optional description that expands on the purpose of the field or prompts a specific action.
+- `step`: An optional stepping interval to use when using up and down arrows to adjust the value, as well as for validation
+
+## Interface
+
+```typescript
+interface TextConfig {
+  name: string
+  component: 'number'
+  label?: string
+  description?: string
+  step?: string
+}
+```
+
+-->
```

Comment:
Is this an intentional arrow? Also looks like next and previous is broken.

Reply:
Nope! good catch

Reply:
It actually looked quite pretty on the preview site 😂